modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -3551,6 +3621,46 @@ private ObjectAction _createObjectAction(String objectActionName) {
 		return objectAction;
 	}
 
+	private int _getObjectEntryVersionsCountByObjectDefinition(
+		long objectDefinitionId) {
+
+		return _objectEntryVersionLocalService.dslQueryCount(
+			DSLQueryFactoryUtil.countDistinct(
+				ObjectEntryVersionTable.INSTANCE.objectEntryVersionId
+			).from(
+				ObjectEntryVersionTable.INSTANCE
+			).innerJoinON(
+				ObjectEntryTable.INSTANCE,
+				ObjectEntryTable.INSTANCE.objectEntryId.eq(
+					ObjectEntryVersionTable.INSTANCE.objectEntryId)
+			).where(
+				ObjectEntryTable.INSTANCE.objectDefinitionId.eq(
+					objectDefinitionId)
+			));
+	}
+
+	private long _getSharingEntriesCountByObjectDefinition(
+			ObjectDefinition objectDefinition)
+		throws Exception {
+
+		ActionableDynamicQuery actionableDynamicQuery =
```

Comment:
Can you change this logic to use the `countByC_CN(long companyId, long classNameId)` you just created with the new finder. You will need to expose this in the service layer.

Reply:
I can implement that change. I haven't done it yet because I'm unsure about adding new methods in the local service strictly for testing purposes. Could we discuss if there is an alternative approach or if this is acceptable?